crates/bevy_ecs/src/event.rs

```diff
@@ -171,6 +176,7 @@ struct EventInstance<E: Event> {
 /// [Example usage standalone.](https://github.com/bevyengine/bevy/blob/latest/crates/bevy_ecs/examples/events.rs)
 ///
 #[derive(Debug, Resource)]
+#[cfg_attr(feature = "bevy_reflect", derive(Reflect))]
 pub struct Events<E: Event> {
     /// Holds the oldest still active events.
     /// Note that `a.start_event_count + a.len()` should always be equal to `events_b.start_event_count`.
```

Comment:
Is it intended to expose the `events_a`/`events_b` split of the `Events` resource and the internal `EventSequence` type?

Reply:
This seemed like the least intrusive solution at the time. This doesn't seem like a particularly dangerous implementation detail to provide backdoor access to, while ensuring that inspectors still work easily with events.